frontend/src/pages/Login.tsx

```diff
@@ -59,7 +59,7 @@ export default function Login() {
     formData.append('password', password);
 
     const jsonPromise = apiClient.passwordAuth(formData);
-    handleAuth(jsonPromise);
+    await handleAuth(jsonPromise);
```

Comment:
`handleAuth` in `handleHeaderAuth` is promise too. May you also check this scenario, please?